request-cert/k8s_certificates.go

```diff
@@ -58,11 +83,9 @@ func kubernetesSigner(csrName string, csr []byte, wantServerAuth bool) ([]byte,
 
 	// Build the certificate signing request.
 	req := &certificates.CertificateSigningRequest{
-		ObjectMeta: types.ObjectMeta{
-			Name: csrName,
-		},
+		TypeMeta:   types.TypeMeta{Kind: "CertificateSigningRequest"},
+		ObjectMeta: types.ObjectMeta{Name: csrName},
 		Spec: certificates.CertificateSigningRequestSpec{
-			Groups:  []string{"system:authenticated"},
```

Comment:
yeah, me neither :) I found this on some use case somewhere, but checking the kubernetes source (my best source of documentation) shows that this wasn't necessary.